xsl/xccdf-share.xsl

```diff
@@ -284,7 +284,7 @@ Authors:
         </xsl:choose>
     </xsl:variable>
 
-    <span class="label label-success">Remediation <xsl:value-of select="$fix_type"/>:</span>&#160;&#160;&#160;<a data-toggle="collapse" data-target="#{generate-id($fix)}">(show)</a><br />
+    <span class="label label-success">Remediation <xsl:value-of select="$fix_type"/>:</span>&#160;&#160;&#160;<a data-toggle="collapse" data-target="#{generate-id($fix)}" tabindex="0" role="link" aria-expanded="false" href="#!">(show)</a><br />
```

Comment:
Even maybe:

```suggestion
    <a class="label label-success" data-toggle="collapse" data-target="#{generate-id($fix)}" tabindex="0" role="button" aria-expanded="false" title="Activate to reveal" href="#!">Remediation <xsl:value-of select="$fix_type"/>:</a><br />
```

Reply:
@vojtapolasek Actually I don't know which would be better in this case `aria-label` or `title`.

Reply:
This should look like
![Screenshot from 2020-01-08 12-15-16](https://user-images.githubusercontent.com/429134/71980126-a01f4400-321f-11ea-9f83-b0d4e4d1b40f.png)